test/test_linalg.py

```diff
@@ -5925,47 +5970,6 @@ def tracker(worker):
 ---(input size: {:4}, eigenpairs:{:2}, units: relative error, maxiter={:4})---
 '''.format(tol, eq_err, eq_err_general, iters1, eq_err_scipy, eq_err_general_scipy, iters2, m, k, niter))
 
-    def _test_addmm_addmv(self, f, t, m, v, *, alpha=None, beta=None, transpose_out=False, layout=torch.strided):
```

Comment:
If guess if you wanted to keep this as a method of TestLinalg you could turn it into a classmethod

Reply:
I want to import this function in test_sparse_csr.py